packages/components/src/test/properties/inputs/UniquePropertyValuesSelector.test.tsx

```diff
@@ -772,6 +773,93 @@ describe("UniquePropertyValuesSelector", () => {
 
       // expect "clear" button to not be shown
       await waitFor(() => expect(container.querySelector(".clear-indicator")).to.be.null);
+
+      const closeFilteringDialogButton = container.querySelector(".presentation-instance-filter-dialog-close-button");
+      await user.click(closeFilteringDialogButton!);
```

Comment:
Is this needed?

Reply:
No, I missed this when removing the others, however it turns out that without this other tests fail. I fixed the setup in this test because the new changes were causing it to load infinite number of pages and that resolved all the issues. Not sure how or why it was causing other tests to fail.